test/Interop/Cxx/extern-var/extern-var.swift

```diff
@@ -0,0 +1,59 @@
+// RUN: %empty-directory(%t)
+// RUN: %target-clang -c %S/Inputs/extern-var.cc -I %S/Inputs -fPIC -o %t/extern-var.o
+// RUN: %target-build-swift %s -I %S/Inputs -o %t/extern-var %t/extern-var.o -Xfrontend -enable-cxx-interop
+// RUN: %target-codesign %t/extern-var
+// RUN: %target-run %t/extern-var
+//
+// REQUIRES: executable_test
+
+import ExternVar
+import StdlibUnittest
+
+var ExternVarTestSuite = TestSuite("ExternVarTestSuite")
+
+ExternVarTestSuite.test("read") {
+  expectEqual(0, counter)
```

Comment:
Consider calling `setCounterFromCxx` first to make tests order-independent.

Reply:
Removed the test (write-from-cxx and write-from-swift cover the behavior already).